tests/CognitiveComplexityTest.elm

```diff
@@ -611,6 +611,41 @@ fun5 n =
                             "fun5": 1
                           }
                        }"""
+        , test "recursive call complexity should not depend on alphabetical order" <|
+            \() ->
+                """module A exposing (..)
+
+b = b
+
```

Comment:
I don't think this compiles, so I don't know how good of a test case this is 🤔

Reply:
Now it does.

Reply:
👍